src/dbt/kipptaf/models/schoolmint/grow/intermediate/int_schoolmint_grow__observations.sql

```diff
@@ -12,6 +11,8 @@ select
     o.academic_year,
     o.is_published,
 
+    oon.notes_text as observation_notes,
```

Comment:
rename `magic_notes_text`

Reply:
the alias or the original column name?

Reply:
the original column name should be `text`, the alias should be `magic_notes_text`. downstream from here, feel free to rename to whatever, but on staging/int I like to preserve the original names as much as possible